yarn.lock

```diff
@@ -4383,6 +4383,11 @@
   resolved "https://registry.yarnpkg.com/@parcel/rust/-/rust-2.10.2.tgz#f574040b298926062d80ac68e8ced3695f9593c9"
   integrity sha512-v/Cyf3iXlzSc6vgvPiEZzqdKAZ1jJ/aZX7y1YSupDh3RoqJI2bZ93kAOyEi+S7P3kshJkQM0px3YveJFOAMUOA==
 
+"@parcel/rust@2.10.1":
```

Comment:
this change shouldn't be here, could you revert this file to main, delete your node_modules, and reinstall? it should not re-appear